src/hooks/useOnOutsideClick.tsx

```diff
@@ -47,12 +65,19 @@ export const useOnOutsideClick = ({containerRef, ignoreClickRefs, onClickOutside
     },
     [onClickOutside, containerRef, ignoreClickRefs]
   )
-
   useEffect(() => {
-    // use capture to ensure we get all events
-    document.addEventListener('mousedown', onOutsideClickInternal, {capture: true})
+    if (handlers.length === 0) {
+      document.addEventListener('click', handleClick)
+    }
+    setTimeout(() => handlers.push(onOutsideClickInternal))
```

Comment:
😭 This is the only way I can get things to work - otherwise the outside click event for the DropdownModal fires at the same time that the dropdown opens.